### PR TITLE
feat: trigger all suites in all folders

### PR DESCRIPTION
**Pull request by sincraianul**
I've reorganized the suites into folders so that they're easier to find/scan:

Screen Shot 2021-12-17 at 16 42 41

Updated the trigger as well to get and execute all suites that are in folders.

### CODE REVIEW

File: api/ghost-inspector/trigger.ts [hunk 1]
```diff
@@ -8,7 +8,91 @@ import fetch from "cross-fetch";
...
+const ghostInspectorResponseSchema = z.object({
+  data: z.array(
+    z.object({
+      _id: z.string(),
+    })
+  ),
+});
+
+/**
+ * Will execute all suites in folders.
```
Comment (osdiab) [inline api/ghost-inspector/trigger.ts:20]: hm I don't think we want it to trigger the shared sequences though?

**Review by osdiab — APPROVED** [review]
wanna clarify that one thing before merging!

File: api/ghost-inspector/trigger.ts (hunk 1 above)
Comment (sincraianul) [reply api/ghost-inspector/trigger.ts:20]: @osdiab I've updated the docs to reflect the new structure and how these are triggered! https://www.notion.so/spinachhr/Browser-Automation-Tests-04b1aaae27424cb09415e30cba654cf2

**Comment by sincraianul** [comment]
Closing because it's not viable to have the tests split up right now.

**Pull request closed without merging**